maven-slf4j-provider/src/main/java/org/slf4j/impl/MavenSimpleLogger.java

```diff
@@ -103,13 +117,9 @@ protected String getLocation(final StackTraceElement e) {
         } else if (e.getFileName() == null) {
             return "Unknown Source";
         } else if (e.getLineNumber() >= 0) {
-            return String.format("%s:%s", e.getFileName(), e.getLineNumber());
+            return e.getFileName() + ":" + e.getLineNumber();
```

Comment:
Shouldn't we go further and pass the message builder as an argument to avoid concatenating strings here ?

Reply:
I think it is not worth the trouble, in the case where the getLocation method would concatenate the file name and the lin e number, we would need to surround `e.getFileName() + ":" + e.getLineNumber()` with the strong style, separately  or with a style set and reset. I see no support for that in the current implementation (style set/reset).

Reply:
feel free to correct me if I am wrong....